Set XDG_MENU_PREFIX to kde-4- if not set. This fixes "empty-applications-menu" when running
KDE4 applications outside of KDE4 environment.

Index: kded/vfolder_menu.cpp
===================================================================
--- a/kded/vfolder_menu.cpp	(revision 1014368)
+++ b/kded/vfolder_menu.cpp	(working copy)
@@ -770,7 +770,9 @@
    QString result;
 
    QString xdgMenuPrefix = QString::fromLocal8Bit(qgetenv("XDG_MENU_PREFIX"));
-   if (!xdgMenuPrefix.isEmpty())
+   if (xdgMenuPrefix.isEmpty())
+      xdgMenuPrefix = "kde-4-";
+
    {
       QFileInfo fileInfo(fileName);
 
